src/crab/type_to_num.cpp

```
@@ -244,24 +288,81 @@ void TypeToNumDomain::havoc_offsets(const Reg& reg) {
+    result.ptr_sum_bindings = intersect_bindings(ptr_sum_bindings, other.ptr_sum_bindings);
+    return result;
 }
 
 TypeToNumDomain TypeToNumDomain::narrow(const TypeToNumDomain& other) const {
-    return TypeToNumDomain{types.narrow(other.types), values.narrow(other.values)};
+    TypeToNumDomain result{types.narrow(other.types), values.narrow(other.values)};
+    result.ptr_sum_bindings = intersect_bindings(ptr_sum_bindings, other.ptr_sum_bindings);
+    return result;
+}
+
+void TypeToNumDomain::bind_ptr_sum(const Reg& intermediate, const Reg& ptr_src, const Reg& num_src,
```

Clang (14.0.6)

[warning] 3 adjacent parameters of 'bind_ptr_sum' of similar type ('const prevail::Reg &') are easily swapped by mistake

(bugprone-easily-swappable-parameters)